### PR TITLE
Close #432 - publishing/archiving behavior is too strict

### LINKED ISSUE #432: pages goes on 404 after opening publish / archive pop in
pages goes on 404 after opening publish / archive pop in

- [x] When publishing/archiving panel is open, date fields should be empty
- [x] If field is emplty, no date should be posted to API

### PR DESCRIPTION
**Pull request by mickaelandrieu**
See https://github.com/backbee/BbCoreJs/issues/432

### CODE REVIEW

File: src/tb/apps/page/templates/elements/scheduling-input.twig [hunk 1]
```diff
@@ -4,15 +4,16 @@
         <div class="input-group input-group-xs">
             <input type="text"
                 name="{{ element.getKey() }}"
+                id="{{ element.getKey() }}"
                 {% if element.isDisabled() %}disabled{% endif %}
                 value="{{ element.getValue() }}"
                 class="form-control input-xs bb5-datepicker"
              >
-            <span class="input-group-btn">
+            <span class="input-group-btn" onclick="document.getElementById('{{ element.getKey() }}').click();">
```
Comment (ndufreche) [inline src/tb/apps/page/templates/elements/scheduling-input.twig]: Never bind directly on the template set it in view please.

Comment (mickaelandrieu) [inline src/tb/apps/page/templates/elements/scheduling-input.twig]: events are not binded with this part, so where should I put this logic ?

File: src/tb/apps/page/views/page.view.contribution.index.js [hunk 2]
```diff
@@ -208,6 +208,22 @@ define(
...
                     }
                 }
+
+                // dispatch click events on datepicker icons
+                jQuery(self.schedulingTag).on('click', '#btn-publishing', jQuery.proxy(
+                    function () {
+                        jQuery('#publishing').trigger('click');
+                    },
+                    self
+                )
```
Comment (mickaelandrieu) [inline src/tb/apps/page/views/page.view.contribution.index.js]: very ugly but jslint enforces me to do this kind of stuff -__-

**Comment by mickaelandrieu** [comment]
ping @ndufreche for review :)

File: src/tb/apps/page/views/page.view.contribution.index.js (hunk 2 above)
Comment (ndufreche) [inline src/tb/apps/page/views/page.view.contribution.index.js]: Ok, I don't know why you have to do that, you can't call directly the function behind #publishing click event but ok if it's work.
In addition, if you don't "this" variable, you don't need use jQuery proxy

Comment (mickaelandrieu) [inline src/tb/apps/page/views/page.view.contribution.index.js]: without the jQuery.proxy() the event wasn't dispatched (and I don't know why), I accept any help :)

Comment (ndufreche) [inline src/tb/apps/page/views/page.view.contribution.index.js]: ok

**Pull request merged**